include/fiction/algorithms/simulation/sidb/is_ground_state.hpp

```diff
@@ -5,45 +5,66 @@
 #ifndef FICTION_IS_GROUND_STATE_HPP
 #define FICTION_IS_GROUND_STATE_HPP
 
-#include "fiction/algorithms/simulation/sidb/minimum_energy.hpp"
-#include "fiction/algorithms/simulation/sidb/quickexact.hpp"
-#include "fiction/algorithms/simulation/sidb/quicksim.hpp"
+#include "fiction/algorithms/simulation/sidb/determine_groundstate_from_simulation_results.hpp"
 #include "fiction/algorithms/simulation/sidb/sidb_simulation_result.hpp"
 #include "fiction/traits.hpp"
-#include "fiction/utils/math_utils.hpp"
 
-#include <cmath>
+#include <cstdint>
+#include <unordered_set>
 
 namespace fiction
 {
 
 /**
- * This function checks if the ground state is found by the *QuickSim* algorithm.
+ * This function checks if the ground state is found by the heuristic.
```

Comment:
Could you please expand this docstring? What ground state? Of what? By what heuristic?